### ISSUE TITLE
Unable to comment/uncomment non-Python file

### ISSUE THREAD

**Issue by Mark531** [spyder-ide/spyder#3500]
## Description of your problem

**What steps will reproduce the problem?**

1. With Spyder 3.0, open a text file (not a *.py file)
2. Select a few lines, right-click and choose comment/uncomment

**What is the expected output? What do you see instead?**
Nothing happens. On previous versions of Spyder, commenting did work on non-Python files.

**Please provide any additional information below**

## Versions and main components

* Spyder Version: 3.0
* Python Version: 3.5
* Operating system: Win 2010

## Dependencies

Please go to the menu entry `Help > Optional Dependencies` (or
`Help > Dependencies`), press the button `Copy to clipboard`
and paste the contents below:
jedi >=0.8.1     :  0.9.0 (OK)
matplotlib >=1.0 :  1.5.1 (OK)
nbconvert >=4.0  :  4.2.0 (OK)
numpy >=1.7      :  1.10.4 (OK)
pandas >=0.13.1  :  0.18.0 (OK)
pep8 >=0.6       :  1.7.0 (OK)
pyflakes >=0.6.0 :  1.3.0 (OK)
pygments >=2.0   :  2.1.3 (OK)
pylint >=0.25    :  1.6.4 (OK)
qtconsole >=4.2.0:  4.2.1 (OK)
rope >=0.9.4     :  0.9.4-1 (OK)
sphinx >=0.6.6   :  1.4.8 (OK)
sympy >=0.7.3    :  1.0 (OK)

**Comment by ccordoba12** [comment]
@goanpeca, what we could about this one?

**Comment by goanpeca** [comment]
I don't understand the issue...

**Comment by ccordoba12** [comment]
Well, now that we are able to highlight all kinds of files, we should also be able to add comments to them :-)

**Comment by Mark531** [comment]
@goanpeca Exactly, this "commenting" feature of Spyder was useful when working for instance on config files. I hope you'll reintegrate this feature in a future Spyder version.

**Comment by goanpeca** [comment]
@Mark531 we will integrate comments for all the available languages, it is on the roadmap ;-)

**Comment by Mark531** [comment]
Great! Thank you.

**Comment by ccordoba12** [comment]
@Mark531, please leave it open so we don't forget to work on it in the future :-)
